src/main/java/loqor/ait/tardis/data/FuelHandler.java

```diff
@@ -15,94 +15,96 @@
 import loqor.ait.tardis.data.properties.doubl3.DoubleValue;
 import loqor.ait.tardis.data.travel.TravelHandler;
 import loqor.ait.tardis.data.travel.TravelHandlerBase;
-import loqor.ait.tardis.wrapper.server.ServerTardis;
 
 public class FuelHandler extends KeyedTardisComponent implements ArtronHolder, TardisTickable {
 
     public static final double TARDIS_MAX_FUEL = 50000;
 
-    private static final DoubleProperty FUEL = new DoubleProperty("fuel_count", 1000d);
-    private static final BoolProperty IS_REFUELING = new BoolProperty("is_refueling", false);
+    private static final DoubleProperty FUEL = new DoubleProperty("fuel", 1000d);
```

Comment:
will stale FuelHandlers lose all their fuel since the key has been changed?

Reply:
No one cares about .5 users
They dont exist i think

Reply:
thats true tbh
the old versions arent real just delete ur world

Reply:
Fr fr